libs/sql-schema-describer/src/lib.rs

```diff
@@ -467,9 +467,33 @@ pub struct Sequence {
     pub allocation_size: u32,
 }
 
+#[derive(PartialEq, Debug, Clone)]
+pub struct DefaultValue {
+    kind: DefaultKind,
+    constraint_name: Option<String>,
+}
+
+impl Serialize for DefaultValue {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        self.kind.serialize(serializer)
+    }
+}
+
+impl<'de> Deserialize<'de> for DefaultValue {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: serde::Deserializer<'de>,
+    {
+        Ok(Self::new(DefaultKind::deserialize(deserializer)?))
+    }
+}
```

Comment:
@do4gr with the release, we won't use the serialize/deserialize impls for SqlSchema anymore. Is there any other use for these implementations, or can we completely remove them?

Reply:
I'd be more than happy to remove them!

Reply:
Yeah, I also looked into that when checking ways to reduce the binary size. Let's discuss that on the call

Reply:
We'll do this in another pull request.